setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
Hacked around a bit and got something close to working in PR ( https://github.com/pyFFTW/pyFFTW/pull/161 ). However, I think have the docs mixed with the source code is causing problems, would propose we split them out into a separate `docs` (or otherwise named) directory separate from the source.